lmdeploy/pytorch/models/deepseek_v2.py

```diff
@@ -656,6 +658,13 @@ def prepare_inputs_for_generation(
         input_ids = context.input_ids
         position_ids = context.position_ids
         attn_metadata = context.attn_metadata
+        if context.is_decoding is True:
```

Comment:
This is cuda-only, we should update in https://github.com/InternLM/lmdeploy/blob/8549698b899f6bfd1bc59e85864de2cfab1cce11/lmdeploy/pytorch/backends/cuda/op_backend.py#L107